llvm.yaml

```diff
@@ -0,0 +1,830 @@
+        -DLLVM_BUILD_EXTERNAL_COMPILER_RT=ON \
+        -DLLVM_BUILD_LLVM_DYLIB=ON \
+        -DLLVM_BUILD_TESTS=OFF \
+        -DLLVM_INCLUDE_TESTS=OFF \
+        -DLLVM_DEFAULT_TARGET_TRIPLE="${{build.arch}}-unknown-linux-gnu" \
+        -DLLVM_ENABLE_ASSERTIONS=OFF \
+        -DLLVM_ENABLE_FFI=ON \
+        -DLLVM_ENABLE_LIBCXX=ON \
+        -DLLVM_ENABLE_PIC=ON \
+        -DLLVM_ENABLE_PROJECTS="clang;lld;clang-tools-extra" \
+        -DLLVM_ENABLE_RUNTIMES="compiler-rt;libcxx;libcxxabi;libunwind" \
+        -DLLVM_ENABLE_RTTI=ON \
+        -DLLVM_ENABLE_SPHINX=OFF \
+        -DLLVM_ENABLE_TERMINFO=ON \
+        -DLLVM_ENABLE_ZLIB=ON \
+        -DLLVM_INSTALL_UTILS=ON \
+        -DLLVM_HOST_TRIPLE="${{build.arch}}-unknown-linux-gnu" \
+        -DLLVM_INCLUDE_EXAMPLES=OFF \
+        -DLLVM_LINK_LLVM_DYLIB=ON \
+        -DLLVM_APPEND_VC_REV=OFF \
+        -DLLVM_INCLUDE_BENCHMARKS=OFF \
+        -DCLANG_CONFIG_FILE_SYSTEM_DIR=/etc/clang-${{vars.major-version}} \
+        -DCLANG_ENABLE_ARCMT=ON \
+        -DCLANG_ENABLE_STATIC_ANALYZER=ON \
+        -DCLANG_LINK_CLANG_DYLIB=ON \
+        -DCLANG_PLUGIN_SUPPORT=ON \
+        -DCLANG_PYTHON_BINDINGS_VERSIONS="$python_version" \
+        -DCLANG_VENDOR=Wolfi \
+        -DLIBCLANG_BUILD_STATIC=ON \
+        -DLIBUNWIND_HAS_NODEFAULTLIBS_FLAG=OFF \
+        -DCOMPILER_RT_INCLUDE_TESTS=OFF \
+        -DCOMPILER_RT_BUILD_SANITIZERS=ON \
+        -DCOMPILER_RT_INSTALL_PATH="${{vars.clang-prefix}}" \
+        -DCOMPILER_RT_BUILD_GWP_ASAN=OFF
+
+      cmake --build output --target llvm-tblgen
+      cmake --build output --target clang-tblgen
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/usr/lib/cmake
+      ln -s ${{vars.llvm-prefix}}/lib/cmake/llvm ${{targets.contextdir}}/usr/lib/cmake/llvm
+
+  - uses: strip
+
+subpackages:
+  - name: libLLVM
+    description: LLVM ${{vars.major-version}} runtime library
+    dependencies:
+      provides:
+        - libLLVM-${{vars.major-version}}=${{package.full-version}}
+    pipeline:
+      - runs: |
+          soname="libLLVM.so.${{vars.major-minor-version}}"
+          sonamefull="libLLVM-${{vars.major-version}}.so"
+          mkdir -p ${{targets.contextdir}}/${{vars.llvm-prefix}}/lib/
+          mv ${{targets.destdir}}/${{vars.llvm-prefix}}/lib/$soname ${{targets.contextdir}}/usr/lib/
+          ln -s $soname ${{targets.contextdir}}/usr/lib/$sonamefull
+          ln -sf ../../$soname ${{targets.destdir}}/${{vars.llvm-prefix}}/lib/$soname
+          ln -sf ../../$soname ${{targets.destdir}}/${{vars.llvm-prefix}}/lib/$sonamefull
+    test:
+      pipeline:
+        - uses: test/ldd-check
+          with:
+            packages: $(basename ${{targets.contextdir}})
+
+  - name: libclang-cpp
+    description: Clang ${{vars.major-version}} runtime library
+    dependencies:
+      provides:
+        - libclang-cpp-${{vars.major-version}}=${{package.full-version}}
+      runtime:
+        - libLLVM
```

Comment:
We've seen some issues with abseil-cpp that leads us to believe depending on the specific version would be the safest thing e.g. `${{package.full-version}}` . See https://github.com/chainguard-dev/internal-dev/issues/9394#issuecomment-2688648434 for an example.

Reply:
Will do!